reporting page

orbeckst opened the pull request:
- COC violation reporting
- ombudspersons

orbeckst committed:
draft for reporting page

orbeckst commented:
@jennaswa and @IAlibay can I list your discord handles for people to DM you? If yes, can you please tell me which one to choose?

I also need to link it from the README.

If you have any other suggestions what should go in there please add/comment.

orbeckst committed:
breach -> violation

IAlibay commented:
> @jennaswa and @IAlibay can I list your discord handles for people to DM you? If yes, can you please tell me which one to choose?
> 
> I also need to link it from the README.
> 
> If you have any other suggestions what should go in there please add/comment.

Yes please! I think highspeedmode is my discord name?

And thanks, will review a bit later.

orbeckst committed:
OB: no SMS

IAlibay committed:
Add Irfan's discord handle

orbeckst committed:
link REPORTING from README

- add reporting paragraph to README
- make clear that CoC applies to online and in-person and Discord/Discussions

orbeckst committed:
add @jennaswa discord handle (tentative)

orbeckst commented on REPORTING.md:
```diff
@@ -0,0 +1,42 @@
+  
+- Jenna Swarthout Goddard (online)
+  - email: <community@mdanalysis.org>
+  - discord: `@jennaswa`
```
@jennaswa I added your personal discord handle — is this ok?

orbeckst committed:
made @IAlibay also online contact point because of TZ difference to @jennaswa

orbeckst commented:
I'll merge this for now – if necessary we change the live page.
